.github/workflows/testing.yml

```diff
@@ -74,7 +74,24 @@
            --cov-append \
            --cov-report term-missing \
            --cov-report xml . \
-           --env_type ${{ matrix.environment-file }}
+           --env_type ${{ matrix.environment-file }} \
+           -m "not wuhan"
+
+       - name: run tests (Wuhan)
+         run: |
+           pytest \
+           neatnet/ \
```

Comment:
```suggestion
           neatnet/tests/test_simplify.py::test_simplify_network_wuhan \
```

Reply:
I think we only really want to run the `test_simplify_network_wuhan()` test, right? Or are we wanting to run all tests + the Wuhan stuff?

Reply:
Just that one but the marking should take care of that.